### DISCUSSION TITLE
Hey Hit our survey, and thanks to you, we'll pitch until €1000 to Techfugees

### DISCUSSION THREAD

**Discussion by SebConejo** [mnfst/llm-gateway#104] [Announcements]
<img src="https://github.com/casejs/CASE/assets/11723962/e88ae33b-d6ef-43c1-8687-f4bf05f0e1eb" width="450px">

Hello community!

🚀 We've just published a survey and would love to hear your responses.

For each survey completed, we're donating €3 to **Techfugees** to empower refugees through tech. 

**The survey should take 3 minutes. It's here:** 👉 https://forms.gle/bbEeTHagNtXhnye48 

**Deadline:** 23/02/2024 🌟

many thanks 🙏🙏🙏

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
